externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/sql/kyuubi/SparkDatasetHelper.scala

```diff
@@ -197,8 +190,7 @@ object SparkDatasetHelper extends Logging {
 
   private def doCommandResultExec(command: SparkPlan): Array[Array[Byte]] = {
```

Comment:
this method could be further changed to 
```suggestion
  private def doCommandResultExec(commandResult: CommandResultExec): Array[Array[Byte]] = {
```

Reply:
one more place: SPARK-35798 also applicable for `doLocalTableScan`

Reply:
`doCommandResultExec` and `doLocalTableScan` are similar now, do the common parts need to be extracted?

Reply:
the current state is fine